src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,75 @@
+package lotto.controller;
+
+import java.util.List;
+import java.util.Map;
+import lotto.vo.BonusNumber;
+import lotto.view.Input;
+import lotto.domain.LottoBuyer;
+import lotto.service.LottoService;
+import lotto.domain.LottoWinningRanks;
+import lotto.view.Output;
+import lotto.vo.PurchaseAmount;
+import lotto.vo.WinningNumber;
+
+public class LottoController {
+    private final LottoService lottoService;
+
+    public LottoController(LottoService lottoService) {
+        this.lottoService = lottoService;
+    }
+
+    public void run() {
+        PurchaseAmount purchaseAmount = getPurchaseAmount();
+        LottoBuyer lottoBuyer = createLottoBuyerWithLottos(purchaseAmount);
+        WinningNumber winningNumber = getWinningNumber();
+        BonusNumber bonusNumber = getBonusNumber(winningNumber);
+
+        Output.printWinningResultsHeader();
+        Output.printLine();
+
+        List<LottoWinningRanks> lottoWinningRanks = getLottoWinningRanks(lottoBuyer, winningNumber, bonusNumber);
+        Map<LottoWinningRanks, Integer> rankCounts = aggregateLottoRanks(lottoWinningRanks);
+
+        printLottoRankResults(rankCounts);
+        printRateOfReturn(lottoWinningRanks, lottoBuyer);
+    }
+
+    private PurchaseAmount getPurchaseAmount() {
+        Output.printPurchaseAmountRequestMessage();
+        return Input.getPurchaseAmount();
+    }
+
+    private LottoBuyer createLottoBuyerWithLottos(PurchaseAmount purchaseAmount) {
+        LottoBuyer lottoBuyer = lottoService.getLottoBuyer(purchaseAmount);
+        Output.printNumberRequestMessage(lottoBuyer.getLottoCount());
+        Output.printLottos(lottoBuyer.getLottos());
+        return lottoBuyer;
+    }
+
+    private WinningNumber getWinningNumber() {
+        Output.printWinningNumberRequestMessage();
+        return Input.getWinningNumber();
+    }
+
+    private BonusNumber getBonusNumber(WinningNumber winningNumber) {
+        Output.printBonusNumberRequestMessage();
+        return Input.getBonusNumber(winningNumber);
+    }
+
+    private List<LottoWinningRanks> getLottoWinningRanks(LottoBuyer lottoBuyer, WinningNumber winningNumber,
+                                                         BonusNumber bonusNumber) {
```

Comment:
이렇게 줄을 바꿔주면 코드가 더 깔끔해질것 같아요!
```suggestion
    private List<LottoWinningRanks> getLottoWinningRanks(
          LottoBuyer lottoBuyer,
          WinningNumber winningNumber,
          BonusNumber bonusNumber
) {
```

Reply:
120자만 넘지만 않게 신경썼는데, 매개변수 부분을 다 내리는게 가독성이 더 좋아보이네요! 감사합니다!